jib-core/examples/README.md

```diff
@@ -2,7 +2,7 @@
 
 Please [file an issue](/../../issues/new) if you find any problems with the examples or would like to request other examples.
 
-For examples on using Jib plugins, see [examples](/../../examples).
+For examples on using Jib plugins, see [examples](../examples).
```

Comment:
Sorry, this one still needs two '/..' I think (the other one did since it was in jib/examples and this one is in jib/jib-core/examples)

Reply:
Oh you're faster than me haha, nevermind.